data/i18n/en/en.toml

```diff
@@ -206,10 +206,7 @@ other = "Objectives"
 other = "Options"
 
 [outdated_blog__message]
-other = "The Kubernetes project considers this article to be outdated because it is more than one year old. Check that the information in the page has not become incorrect since its publication."
-
-[outdated_blog__title]
-other = "Outdated article"
+other = "This article is more than one year old. Older articles may contain outdated content. Check that the information in the page has not become incorrect since its publication."
```

Comment:
Hi @liggitt
The original text indicates clearly that the Kubernetes project has marked the article as outdated (not necessarily incorrect). The updated text is vague about whether the content is outdated.

Suggest to edit:
This article is more than one year old and may contain information that is not current. Check that the content ...

Reply:
But we *haven't* marked the particular article as outdated… indicating the possibility seems preferable to saying it *is* outdated

Reply:
The point is that we don't know that every article written more than a year ago is outdated, but the header was asserting they all were.